scripts/deploy-with-s3.js

```diff
@@ -124,7 +124,7 @@ async function main() {
   await remove(publicDir)
 }
 
-main().catch(e => {
+main().catch((e) => {
```

Comment:
should we just simplify this rule? looks like consistency breaks readability in simple cases cc @iAdramelk @fabiosantoscode ?

Reply:
Prettier's team changed `arrowParens` to `always` in v2. I manually set it to the previous one `avoid` and returned back formatting